### ISSUE TITLE
physicsFreeze always disabled

### ISSUE THREAD

**Issue by M1904** [SBPrime/AsyncWorldEdit#371]
Hello SBPrime,

I don't know exactly since when, but the "physicsFreeze" function has not worked for a long time. Whenever I copy sand in air, for example, it falls down or water flow away. (I think that the option in the config is responsible for this, correct me if I'm wrong.) In Minecraft 1.12. you could still let sand "fly". Did I choose the wrong setting or is it no longer possible?

Incidentally, there are other problems because of this. When I copy sand or concrete powder, it falls due to the unfavorably chosen block-placement logarithm (was also more server-friendly at the time, from northwest-bottom to southeast-top, without air-block) and drop away.

Server Version: PaperSpigot 1.16.5 (git-Paper-575)
Awe-Premium-Version: 3.8.2
WE-Version: 7.2.4+5723-e3bfc64

**Comment by SBPrime** [comment]
Hey the physics freeze stops blocks from falling >> ONLY << if any of the neighboring blocks are queued for change. Is this your scenario?

To do a proper test you can copy paste a pattern:
`From top to bottom:
Layer 1:
CCCCC
CSCWC
CCCCC

Layer 2:

CCCCC
CCCCC
CCCCC`

Make sure to set AWE speed to 1 block (/awe speed 1)

The block placing order was changed internally in WE, I did not do anything, do you think I should „fix” that in WE?

**Comment by M1904** [comment]
the easiest way to show you the problem: placing sand https://youtu.be/he_AYbGSTkI

I think, it would make sense to fix that.

Sry, but i don't understand your "proper test". How should i do this correctly?
AWE speed 1 is no problem, but what do you mean with the layer?

**Comment by SBPrime** [comment]
let me rephrase: do you see the same problem when you set awe speed to 1.
